src/scripts/create_combined_sic_activites.js

```diff
@@ -8,13 +8,19 @@
   - sic_description (from condensed_sic_codes)
   - is_ch_activity (true if parent is ch_economic_activity_sic_codes)
   - activity_description_lower_case (lower case of the activity_description field)
+  - generation_date has been added to keep track of when records have been created
 
 
 */
 
 
 db.combined_sic_activities.drop()
-db.ons_combined_sic_activities.aggregate([{$out : "combined_sic_activities"}])
+db.ons_combined_sic_activities.aggregate([ {
```

Comment:
could you do this in src/scripts/create_ons_combined_sic_activites.js so that we just have the date when this (ONS) file was generated. I am not expecting this to happen that often (maybe every 3 to 5 years)